_posts/2020-08-31-JabRef5-1..md

```diff
@@ -0,0 +1,44 @@
+---
+title:  JabRef 5.1 Release 
+id: jabref-5-1
+author: "[JabRef Developers](https://github.com/JabRef/jabref/blob/master/DEVELOPERS)" 
+---
+
+We are proud to announce the release the next milestone of JabRef. Today we are releasing Jabref 5.1.
+
+## Notable Changes and New Features
+
+After the release of the previous major milestone 5.0 we intensively focused on performance and bug fixes. We could speed up JabRef 5.1 significantly, especially when used with many groups and or larger libraries.
+
+We didn't just focus on performance and bug fixes, but also on a couple of new features. For users using our integrated fetchers we now added the option to automatically download the associated fulltext (if avaiable) on import.
+Another related feature ist that now also supports retrieving metadata from [mEDRA DOIs](https://www.medra.org/).
+We also improved the integrity check which can now detect dupliate DOIs which is a reliable way to check for duplicates in the libary.
+Despite these improvements under the hood we also tweaked the iser interface and added ooltips for nearly all fields in the entry editor to show a description of the fields purpose.
+
+One major change affects users of Mac computers running Mac OS Catalina. You will no longer get any warnings about insecure software. JabRef for mac is now completely signed and notarized. We also took the opprutnity to adjust the dmg background and add the JabRef icon to the installer.
+
+Despite some bug fixes under the hood we also improved the user interface. The entry editor now supports markdown in the comment field.
+We alos added a progress-indicator showing the average progress of background tasks to the toolbar. This gives you an overview of running background tasks.
+Another change is related to the preferneces dialog, we refined the structure and streamlined the user exeprience.
+
+For LaTeX users, JaRef now supports the entry types and fields from `biblatex-software` out of the box.
+
+[And many more! In total, there are over 100 changes](https://github.com/JabRef/jabref/blob/master/CHANGELOG.md) since our last offical release!
+
+**You can get it from [FOSShub](https://www.fosshub.com/JabRef.html).**
+
+Please report any other issues that you encounter in our [bug tracker](https://github.com/JabRef/jabref/issues) (in case it was not already reported before).
+
+## Breaking Changes
```

Comment:
This section can be removed, right? I'm not aware of any breaking changes from 5.0 to 5.1.

Reply:
Removed it. The groups format change happened long time ago.